docs/src/app/components/pages/components/Chip/ExampleArray.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
```

Comment:
I noticed this too but saw that all the examples haven't been migrated yet. Maybe we could do them all in a separate PR? Doesn't matter to me though